packages/new-polymath-issuer/src/pages/DividendsWizard/Step-3/index.tsx

```diff
@@ -255,18 +255,43 @@ const Step3Base: FC<Props> = ({
         }}
         validate={handleValidation}
         onSubmit={onSubmit}
-        render={({ values }) => {
+        render={({
+          values,
+          setFieldTouched,
+          setFieldError,
+          isValid,
+          initialValues,
+          touched,
+        }) => {
           const { currency } = values;
-
           return (
             <form
               onSubmit={submitEvent => {
-                submitEvent.persist();
                 submitEvent.preventDefault();
-                setFormSubmissionStatus({
-                  isSubmitting: true,
-                  submitEvent,
-                });
+                for (const key of Object.keys(initialValues || {})) {
+                  if (Object.keys(touched).indexOf(key) === -1) {
+                    if (
+                      key !== 'tokenAddress' ||
+                      currency === types.Tokens.Erc20
+                    ) {
+                      setFieldTouched(`${key}`, true);
+                    }
+                  }
+                }
+                if (
+                  values.dividendAmount &&
+                  values.dividendAmount.isEqualTo(new BigNumber(0))
```

Comment:
Why do `new BigNumber(0)` instead of just `=== 0`?

Reply:
Doing `=== 0` gives an error that the statement will always resolve to `false`:

<img width="615" alt="Screen Shot 2019-03-19 at 8 04 28 AM" src="https://user-images.githubusercontent.com/13947919/54604690-c8824b80-4a1d-11e9-8cc5-74bd58f229c8.png">

Reply:
Changed to Formik validation